docs/content/installation/uninstall_driver_openshift_web.md

```diff
@@ -11,12 +11,18 @@ Perform the following steps in order to uninstall the CSI driver and operator th
 
     **Operators** > **Installed Operators** > **Operator Details**.
 
-4.  Click on the **more** menu for the **ibm-block-csi** driver and select **Delete IBMBlock CSI**.
+4. If applicable, click on the **more** menu for the **host-definer** plugin and select **Delete HostDefiner**.
```

Comment:
```suggestion
4. If applicable, click on the **more** menu for **host-definer** and select **Delete HostDefiner**.
```
or if we must have a type:
```suggestion
4. If applicable, click on the **more** menu for the **host-definer** component and select **Delete HostDefiner**.
```

Reply:
@oriyarde  I took the terminology from the UI itself (there is a screen capture in the ticket - image (14).png)

Reply:
I couldn't find the word "plugin" in `image (14).png` - could you capture a smaller area from it and put it here/in the ticket?

Reply:
My mistake - it's "driver". i'll update accordingly

![image](https://user-images.githubusercontent.com/54570654/178147101-3905ba50-4261-45ea-b2f6-911916a43ec1.png)

Reply:
I'm pretty sure this outdated as well.
@matancarmeli7 ?

Reply:
I am not sure what do you mean?

Reply:
@matancarmeli7 is this the latest screen capture with the term "driver" in the tab name?

Reply:
yes

Reply:
updated to driver